test/jasmine/tests/download_test.js

```diff
@@ -0,0 +1,88 @@
+var Plotly = require('@lib/index');
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var textchartMock = require('../../image/mocks/text_chart_arrays.json');
```

Comment:
use `@mocks/text_chart_arrays.json` instead of `../../image/mocks/text_chart_arrays.json`.